test/parallel/test-process-argv-0.js

```diff
@@ -14,15 +9,10 @@ if (process.argv[2] !== 'child') {
   });
 
   var childArgv0 = '';
-  var childErr = '';
   child.stdout.on('data', function(chunk) {
     childArgv0 += chunk;
   });
-  child.stderr.on('data', function(chunk) {
-    childErr += chunk;
-  });
   child.on('exit', function() {
```

Comment:
I was actually just going to just change `child.on('exit', ...)` to `process.on('exit', ...)` rather than use `common.mustCall()` for two reasons:

* I think it's clearer what is going on that way.
* This test does not load `common.js` to begin with because it doesn't need it. Loading the entire thing just for one use of `common.mustCall()` seems like overkill when `process.on('exit', ...)` will work just fine.

You (@targos) may very well be correct that `child.on('exit', ...)` and `process.on('exit', ...)` have the exact same behavior. But, at least according to the docs, the exit event on a `ChildProcess` might not fire. I can't find any similar warning in the docs on the exit event for `process`. So I'm inclined to go with `process`. If the `exit` event might not fire, that might be an issue to file against the docs. The text describing `beforeExit` seems to imply that `exit` can be depended on to fire for `process`.